src/api/app/models/configuration.rb

```diff
@@ -97,9 +98,7 @@ def write_to_backend
   end
 
   def invalidate_cache
-    Rails.cache.fetch('configurations/1', expires_in: 1.day, force: true) do
-      self
-    end
+    Rails.cache.delete('configurations/1')
```

Comment:
@eduardoj with the switch from `fetch` to `delete` we will end up with an empty cache. Shouldn't we write it to the cache afterwards?

Reply:
because the fetch used `force` to cause rewrite, this is not given anymore

Reply:
alright